packages/plugin-github/sources/githubUtils.ts

```diff
@@ -1,27 +1,55 @@
+
 const githubPatterns = [
   /^github:([^\/#]+)\/([^\/#]+)(?:#(.+))?$/,
-  /^git@github.com:([^\/#]+)\/([^\/#]+)(?:#(.+))?$/,
+  /^[git@github.com:|git:\/\/github.com]+([^\/#]+)\/([^\/#]+)\.git(?:#(.+))?$/,
   /^git\+https:\/\/github.com\/([^\/#]+)\/([^\/#]+)\.git?(?:#(.+))?$/,
   /^https:\/\/github.com\/([^\/#]+)\/([^\/#]+)\/tarball\/([^\/#]+)(?:#|$)/,
+  /^[http|https]+:\/\/.*github.com\/([^\/#]+)\/([^\/#]+)\.git?(?:#(.+))?$/,
```

Comment:
:heavy_check_mark: 

Thanks for catching that! Using non-capturing groups now `(?:)`

I think http is acceptable because github allows it:
http://github.com/yarnpkg/berry/archive/master.zip